modules/github/widget.go

```diff
@@ -87,6 +129,14 @@ func (widget *Widget) currentGithubRepo() *GithubRepo {
 	return widget.GithubRepos[widget.Idx]
 }
 
+func (widget *Widget) openPr() {
+	currentSelection := widget.View.GetHighlights()
+	if widget.Selected >= 0 && currentSelection[0] != "" {
+		url := (*widget.currentGithubRepo().RemoteRepo.HTMLURL + "/pull/" +  widget.View.GetRegionText(currentSelection[0]))
+		utils.OpenFile(url)
```

Comment:
These changes look great, however this part's going to cause issues. The selected text can be padded with spaces; on my machine the URL ends up being `https://github.com/wtfutil/wtf/pull/%20584`, which is invalid.

If there's a way to do this using `currentGithubRepo()`, then using the selected line index and calling against the repo's `PullRequests` property, that would be preferable.

Reply:
Sounds good to me, I'm thinking of a way I can store all the issue numbers into a slice and reference them using the Selected integer

Reply:
I've fixed this in commit c21b7c32a838f339b806ff886a20a35af8c07230